### ISSUE TITLE
viber doesn't work

### ISSUE THREAD

**Issue by autohitdevman** [uglide/viber-unofficial-snap#15]
Viber didn't work.

"Your number is required for authentification. This number will be used as your ID in Viber"

It means that it is a old version according to support 

Can you check this out?

**Comment by dimige** [comment]
Yes this looks abandoned. There is no new commits after Mar 2019 ...
